### PR TITLE
suggest howto for revhistory entries

### PR DESCRIPTION
**Pull request by janajaeger**
Provide a few tips on how to write meaningful revhistory entries.

### CODE REVIEW

File: templates/articles/assembly.asm.xml [hunk 1]
```diff
@@ -63,15 +63,41 @@
...
-            </para>
+            <itemizedlist>
+              <listitem>
+                <para>
+                  Do not add any links here (no linking to sections, no Bugzilla links, no links at all) 
+                </para>
+              </listitem>
+              <listitem>
+                <para>
+                  Add Bugzilla numbers (not links!) wherever appropriate
```
Comment (tomschr) [inline templates/articles/assembly.asm.xml:78]: I'd advocate to use the [bug abbreviations](https://en.opensuse.org/openSUSE:Packaging_Patches_guidelines#Current_set_of_abbreviations) as done in OBS packaging. For example, `bsc#12345` would refer to our Bugzilla `bugzilla.suse.com` with the bug number 12345. For GitHub, `gh#domain/project#123` etc.

**Review by tomschr — COMMENTED** [review]
Great addition. :+1: I have only one suggestion to clarify _how_ we should refer (not link) to bugs (whatever issue tracker is used).

**Review by dariavladykina — COMMENTED** [review]
Hi, please see a couple of suggestions here. Thanks!

File: templates/articles/assembly.asm.xml (hunk 1 above)
Comment (janajaeger) [reply templates/articles/assembly.asm.xml:78]: Good point!

File: templates/articles/assembly.asm.xml [hunk 2]
```diff
@@ -63,15 +63,44 @@
...
+        <revision><date>2024-11-13</date>
           <revdescription>
-            <para>
-              Describe the purpose of this revision
-            </para>
+            <itemizedlist>
+              <listitem>
+                <para>
+                  Do not add any links here (no linking to sections, no issue 
+                  tracker links, no links at all)
```
Comment (dariavladykina) [inline templates/articles/assembly.asm.xml:74]:
```suggestion
                  tracker links, no links at all).
```

**Review by dariavladykina — COMMENTED** [review]
Just a tiny nit from me. Thanks!

**Comment by dariavladykina** [comment]
Looking at it again, I would suggest changing the order of paragraphs like this:
1. Assign a paragraph for each type
2. Add one paragraph per Bugzilla issue
3. Add issue numbers (not links!) 
4. Do not add any

**Comment by janajaeger** [comment]
Template changes should only be made inside doc-kit. This has been applied there.

**Comment by tomschr** [comment]
For the record, it's moved to openSUSE/doc-kit#84

**Pull request closed without merging**